Packs/CircleCI/Integrations/CircleCI/CircleCI.py

```diff
@@ -0,0 +1,302 @@
+        - 'limit' (int): Maximum number of results to return.
+
+    Returns:
+        (CommandResults).
+    """
+    vc_type, organization, project, limit = get_common_arguments(client, args)
+    job_number: str = args.get('job_number', '')
+    artifact_suffix: Optional[str] = args.get('artifact_suffix')
+
+    response = get_response_with_pagination(client.get_job_artifacts, [vc_type, organization, project, job_number],
+                                            limit)
+
+    if artifact_suffix:
+        response = [artifact for artifact in response if artifact.get('path', '').endswith(artifact_suffix)]
+    else:
+        response = response[:limit]
+
+    return CommandResults(
+        outputs_prefix='CircleCI.Artifact',
+        outputs_key_field='url',
+        readable_output=tableToMarkdown('CircleCI Artifacts', response, removeNull=True,
+                                        headerTransform=camelize_string),
+        outputs=response
+    )
+
+
+def circleci_workflow_jobs_list_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Retrieve jobs list from CircleCI workflow.
+    Args:
+        client (Client): Client to perform the API calls
+        args (Dict[str, Any]): XSOAR arguments.
+        - 'workflow_id' (str): Workflow ID to retrieve its jobs.
+        - 'limit' (int): Maximum number of results to return.
+
+    Returns:
+        (CommandResults).
+    """
+    workflow_id: str = args.get('workflow_id', '')
+    limit = arg_to_number(args.get('limit')) or DEFAULT_LIMIT_VALUE
+    response = get_response_with_pagination(client.get_workflow_jobs, [workflow_id], limit)
+
+    return CommandResults(
+        outputs_prefix='CircleCI.WorkflowJob',
+        outputs_key_field='id',
+        readable_output=tableToMarkdown(f'CircleCI Workflow {workflow_id} Jobs', response, removeNull=True,
+                                        headerTransform=camelize_string),
+        outputs=response
+    )
+
+
+def circleci_workflow_last_runs_command(client: Client, args: Dict[str, Any]) -> CommandResults:
+    """
+    Retrieve jobs list from CircleCI workflow.
+    Args:
+        client (Client): Client to perform the API calls
+        args (Dict[str, Any]): XSOAR arguments.
+        - 'workflow_name' (str): Name of workflow to retrieve its last runs details.
+        - 'vc_type' (str): VC type. One of 'github', 'bitbucket'.
+        - 'organization' (str): Organization to retrieve workflow last runs from.
+                                Defaults to artifacts parameter is none is given.
+        - 'project' (str): Project to retrieve workflow last runs from. Defaults to project parameter is none is given.
+        - 'limit' (int): Maximum number of results to return.
+    Returns:
+        (CommandResults).
+    """
+    vc_type, organization, project, limit = get_common_arguments(client, args)
+    workflow_name: str = args.get('workflow_name', 'nightly')
```

Comment:
```suggestion
    workflow_name: str = args.get('workflow_name')
```
we will set the value in the relevant place (`nightly` is not necessarily a good default value for other customers)

Reply:
It was during my checks, i forgot to remove. my bad